core/domain/voiceover_domain.py

```diff
@@ -185,25 +186,28 @@ def add_voiceover(
         voiceover_type: feconf.VoiceoverType,
         voiceover: state_domain.Voiceover
     ) -> None:
-        """Adds voiceover to the entity voiceover instance."""
-        if not isinstance(voiceover_type, feconf.VoiceoverType):
-            raise utils.ValidationError(
-                'voiceover type must be VoiceoverType, received %s' %
-                voiceover_type)
-        voiceover.validate()
+        """Adds voiceover to the entity voiceover instance.
+
+        Args:
+            content_id: str. The ID of the content.
```

Comment:
Explain how this relates to the function's objective. E.g. The ID of the content for which the voiceover is being added.

Ditto below.

Reply:
Updated